### PR TITLE
feat(cmd): add .devlanguages

### PR DESCRIPTION
**Pull request by NUTTTTTTTT**
## Description
Adds .devlanguages to provide a list of development languages used in the A32NX.

Responds to .devlanguage, .devlanguages, .devlang and .dl.
fixes https://github.com/flybywiresim/discord-bot/issues/481

## Test Results
image

## Discord Username
nuhtt

### CODE REVIEW

File: src/commands/general/devLanguages.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        'The A32NX is built using a number of development languages for different aspects of the aircraft. The below list confirms the current languages used, however is not exhaustive.',
+        '',
+        '**Displays:** Typescript',
+        '**Autopilot/flight controls:** MATLAB/C++',
+        '**Systems/physics simulations:** Rust',
+    ]),
+});
+
+export const devLanguages: MessageCommandDefinition = {
+    name: ['devlanguage', 'devlanguages'],
```
Comment (Sleinmaster) [inline src/commands/general/devLanguages.ts:17]:
```suggestion
    name: ['devlanguage', 'devlanguages','devlang'],
```

Comment (NUTTTTTTTT) [reply src/commands/general/devLanguages.ts:17]: good shout!

File: .github/command-docs.md [hunk 2]
```diff
@@ -100,6 +100,7 @@
 | Command           | Description                                                                                                                                                                                | Alias                                                       |
 |:------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------|
 | .botissue         | Provides details on where to create a FBW Discord Bot Issue or Feature Request                                                                                                             | .bot-issue <br> .botfeature <br> .bot-feature <br> .botfeat |
+| .devlanguages     | Provides a list of development languages used in the A32NX                                                                                                                                 | .devlanguage <br> .devlang <br> .dl                         |
```
Comment (Lauvaan) [inline .github/command-docs.md:103]: Should be before .botissue to be in line with the alphabetical order used in the command-docs.md

**Review by Lauvaan — CHANGES_REQUESTED** [review]

File: .github/command-docs.md (hunk 2 above)
Comment (NUTTTTTTTT) [reply .github/command-docs.md:103]: D comes after B?

Comment (Lauvaan) [reply .github/command-docs.md:103]: Yeah no brain stopped working

File: .github/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+Update <small>_ July 2023</small>
+
+- feat: added .devLanguages to confirm languages used in A32NX.
+
 Update <small>_ June 2023</small>
 
 - feat: added .spad command, which links to the SPAD.neXt doscord server (28/06/2023)
```
Comment (benw202) [inline .github/CHANGELOG.md:9]:
```suggestion
- feat: added .spad command, which links to the SPAD.neXt discord server (28/06/2023)
```

Just spotted this, can fix here

File: .github/command-docs.md
Comment (benw202) [inline .github/command-docs.md:1]: Can I get you to fix Spad here. Line 116. Just missing a `|` at the end

File: src/commands/general/devLanguages.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { MessageCommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+const devLanguagesEmbed = makeEmbed({
+    title: 'FlyByWire A32NX | Development Languages',
+    description: makeLines([
+        'The A32NX is built using a number of development languages for different aspects of the aircraft. The below list confirms the current languages used, however is not exhaustive.',
```
Comment (benw202) [inline src/commands/general/devLanguages.ts:8]:
```suggestion
        'The A32NX is built using a number of development languages for different aspects of the aircraft. The list below confirms the current languages used, however is not exhaustive.',
```

Prefer readability?

**Review by benw202 — CHANGES_REQUESTED** [review]

**Review by benw202 — APPROVED** [review]

**Pull request merged**